src/python/doc/rips_complex_sklearn_itf_ref.rst

```diff
@@ -0,0 +1,54 @@
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+Rips complex persistence scikit-learn like interface
+####################################################
+
+.. list-table::
+   :width: 100%
+   :header-rows: 0
+
+   * - :Since: GUDHI 3.9.0
```

Comment:
Sorry, 3.10 now, my fault :bowing_man:

Reply:
I modified it on [86cfb79](https://github.com/GUDHI/gudhi-devel/pull/691/commits/86cfb7982270e7324803c50f892883ebc5efaa98) and added the feature in the next release note on [31bbd29](https://github.com/GUDHI/gudhi-devel/pull/691/commits/31bbd292bd33cffcc3b06186c935b2e76ad897da)